### ISSUE TITLE
TypeError: forward() got an unexpected keyword argument

### ISSUE THREAD

**Issue by will-mrtr** [haofanwang/ControlNet-for-Diffusers#34]
Have been getting this error every single time, after a great deal of debugging in an effort to get this code to run.
I can not figure out why this error persists, as from what Ive seen, this method is used for every single example of building an inference pipeline with Diffusers.
Some help here would be greatly appreciated because something could be bugged.

<img width="1097" alt="Screenshot 2023-03-16 at 3 51 19 PM" src="https://user-images.githubusercontent.com/112717725/225770259-97bfbfe8-268a-4b99-b71a-69fa1251f70d.png">

**Comment by haofanwang** [comment]
As shown in the README.md, you should use `StableDiffusionControlNetInpaintPipeline`. @will-mrtr

**Comment by will-mrtr** [comment]
Yeah I tried that too, I ran everything as you had it in the readme to begin, doesn't work. 
Diffusers is installed correctly, there might be something wrong with the pipelines - cant even run the pipeline script you gave without it breaking.
Im probably missing something here as well, so some more help would be greatly appreciated.

<img width="1001" alt="Screenshot 2023-03-21 at 9 17 14 AM" src="https://user-images.githubusercontent.com/112717725/226672619-b1b2bb1a-882c-4bdc-a937-02eb61910d71.png">

**Comment by haofanwang** [comment]
It seems that there is no such a pipeline in your env. What is your diffusers version? You should use the provided commit id when installing diffusers. @will-mrtr 

Anyway, I will make this tutorial compatible to the latest diffusers soon.

**Comment by will-mrtr** [comment]
Awesome, thanks for all your help.
I'll reinstall diffusers with the provided commit id

**Comment by will-mrtr** [comment]
Hi there @haofanwang , thanks for your assistance here, but I am still having the same problem after more troubleshooting and following the exact instructions in the README.md

How can I properly import this pipeline in the environment and also into the `__init__.py` files?

For whatever reason, the `StableDiffusionControlNetInpaintPipeline` will not import from the file.
I placed the `__init__.py` files in the two correct paths and also installed `diffusers` from the correct commit id.
I do not know what the problem is.

<img width="1000" alt="Screenshot 2023-03-22 at 3 19 15 PM" src="https://user-images.githubusercontent.com/112717725/227051192-c78669bb-5b77-4a9f-9829-824ce7967f89.png">
